csrc/include/natten/cuda/fna/kernel_forward.h

```diff
@@ -197,6 +197,7 @@ struct FusedNeighborhoodAttentionKernel {
     Dim key_tile_shape;
 
     Dim lse_strideM;
+    Dim maximums_strideM;
```

Comment:
`lse_strideM` is equal to `maximums_strideM` with the conditions we force on these tensors. Remove and just replace with `stats_strideM` with a comment that it applies to both `lse` and `maximums`. Or just use `lse_strideM` for both.